src/java/org/apache/cassandra/transport/Dispatcher.java

```diff
@@ -237,9 +267,12 @@ public static void shutdown()
         {
             requestExecutor.shutdown();
         }
+        if (authExecutor != null)
+        {
+            authExecutor.shutdown();
+        }
```

Comment:
nit: Is it even possible for `requestExecutor` or `authExecutor` to be `null`? (They're both `static final`.)

Reply:
I don't believe it is. Tidied up both new and old executor. 👍